src/kubernetes.cc

```diff
@@ -175,6 +177,7 @@ json::value KubernetesReader::ComputePodAssociations(const json::Object* pod)
         {"topLevelControllerType", json::string(top_level_kind)},
         {"topLevelControllerName", json::string(top_level_name)},
       })},
+      {"nodeName", json::string(node_name)},
```

Comment:
Is it a common convention to use camel casing for metadata labels vs underscores for resource labels? Is there any value in trying to push for consistency?

Reply:
The monitored resource labels use `snake_case`. Since the associations are part of the metadata blobs, we've tried to be consistent with the source of the metadata -- in this case, the Kubernetes API. Kubernetes uses `camelCase`, so we stuck with that.

Reply:
SGTM